src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3338,6 +3338,14 @@ public override void PostVisit(DottedNameNode node)
                     return;
                 }
 
+                // Disable SingleColumnsAccess syntax. I.e. [1,2,3].Values.
```

Comment:
>[1,2,3].Values [](http://example.com/codeflow?start=60&length=14)

We should give a better example, such as `[{a:1,b:2},{a:3,b:4]].a`. Same with the example in the line below.

Reply:
updated thanks!